TACC/Core-CMS-Custom
# Exception list React implementation

## Overview

Exception list React implementation

## Related

[WP-635](https://tacc-main.atlassian.net/browse/WP-635)

## Changes

Exception list and modals conversion.

## Testing

1. Navigate to http://localhost:8000/administration/list-exceptions/
2. Select filters and make sure those filters are applied to the Exceptions list
3. From Actions drop-down select View Record, make sure the information is displayed
4. From Actions drop-down select Edit Record, make sure the information is updated and changes are reflected on View Record modal window and listing

## UI

<img width="1478" alt="Screenshot 2024-10-15 at 2 52 25 PM" src="https://github.com/user-attachments/assets/939b2837-9cfa-44db-ac4d-1a0059982b82">
<img width="798" alt="Screenshot 2024-10-15 at 2 52 42 PM" src="https://github.com/user-attachments/assets/9bc11d6c-02be-443d-82cd-f0235d1a625b">
<img width="484" alt="Screenshot 2024-10-15 at 2 52 58 PM" src="https://github.com/user-attachments/assets/18dbc52f-bda1-4fc3-804d-342f9b5c46df">
<img width="477" alt="Screenshot 2024-10-15 at 2 53 08 PM" src="https://github.com/user-attachments/assets/5a05c6da-d4d0-48f8-95b2-04d2373eaa85">
